documentation/api-scenario/how-to/QuickStart.md

```diff
@@ -1,150 +0,0 @@
-<!--
- Copyright (c) 2021 Microsoft Corporation
-
- This software is released under the MIT License.
- https://opensource.org/licenses/MIT
--->
-
-# Quick start with API Scenario test
-
-## Install
-
-`oav` is an open-source powerful tool for swagger validation, API Scenario testing, and examples generation. GitHub: https://github.com/Azure/oav.
```

Comment:
@scbedd given that you own `oav`: do you recall if it is still being used to power `api-scenario` ? If so, can you have a look on these docs to assess if they are current?

Reply:
Yes I know OAV supports example generation. It is also used to produce coverage reports for specs.

Reply:
@weshaggard so we still use `api-scenario` functionality, yes?

Reply:
In any case, I reverted deletions in `documentation/api-scenario/`.